backend/.idea/workspace.xml

```diff
@@ -99,4 +107,8 @@
       </map>
     </option>
   </component>
+  <component name="VcsManagerConfiguration">
+    <MESSAGE value="add readme.md" />
```

Comment:
Zobacz to jest jakaś historia commitów, pewnie dla `PyCharm`a
<img width="400" alt="image" src="https://user-images.githubusercontent.com/13302050/209391917-19081d1d-c0cb-455c-a58b-b0c46352b78b.png">

Chyba nie ma sensu tego commitować. 🤔

Reply:
Approvuję ✅ , ale spróbujmy się pozbyć tego później, to nie są pliki które chcemy wersjonować

Reply:
to są te instrukcje, a konflikt był w ukrytym pliku workspace.xml